### DISCUSSION TITLE
How about code coverage ?

### DISCUSSION THREAD

**Discussion by ghost** [wjakob/nanobind#244] [Q&A] [answered]
Good morning everyone,

I have a python module written with 100%python parts and a huge chunk actually written in C++ and bound using nanobind.
I am exclusively using the PyTest framework to run tests, and PyCov for the coverage. 

Issue: PyCov is only able to capture/compute the coverage of the code written 100% in Python.

I haven't found anything meaningful here or in the pybind11 discussions about hybrid-coverage like this, so I wanted to ask if anyone had attempted it before and if anyone were successful ?

Thanks guys !
Take good care, 
T.

**Comment by wjakob** [comment, marked as answer]
I haven't used code coverage tools myself. But wouldn't any standard tool for C++ code coverage work to cover the bound parts? (After all, it's just an ordinary C++ lib.)

I don't think there is a tool for dealing with both languages simultaneously.
